README.md

```diff
@@ -194,22 +194,22 @@ With Logging you can also list log entries that have been previously written. Ad
 imports at the top of your file:
 
 ```java
-import com.google.cloud.Page;
+import com.google.api.gax.paging.Page;
 import com.google.cloud.logging.LogEntry;
 import com.google.cloud.logging.Logging.EntryListOption;
 ```
 
 Then, to list the log entries, use the following code:
 
 ``` java
-Page<LogEntry> entries = logging.listLogEntries(
-    EntryListOption.filter("logName=projects/" + options.getProjectId() + "/logs/test-log"));
-Iterator<LogEntry> entryIterator = entries.iterateAll().iterator();
-while (entryIterator.hasNext()) {
-  System.out.println(entryIterator.next());
+Page<LogEntry> entries = logging.listLogEntries(EntryListOption.filter(logFilter));
```

Comment:
ditto

Reply:
I do not change `.md`. It is auto-generated.